testsuites/syncgateway/functional/topology_specific_tests/multiple_accels/test_multiple_accels.py

```diff
@@ -127,7 +127,7 @@ def test_dcp_reshard_sync_gateway_comes_up(params_from_base_test_setup, sg_conf)
         log_info(">>> Adding Seth docs")  # FOX
         futures[executor.submit(seth.add_docs, 4000)] = "seth"
```

Comment:
Remove "seth". Change it to user1 or something like that.

Reply:
done